server.go

```diff
@@ -1034,8 +1034,13 @@ func (s *Server) HandleGetPayloadV2(w http.ResponseWriter, r *http.Request) {
 		Cluster:                   cluster,
 		UserAgent:                 userAgent,
 		SlotUID:                   headerSlotUID,
-	})
-
+	}); err != nil {
+		log.Error().Err(err).Msg("error in GetPayloadV2")
```

Comment:
remove this logging...responsdError would log anyway
```suggestion
		log.Error().Err(err).Msg("error in GetPayloadV2")
```

Reply:
done